test/integration/stomp_/publisher/create_request_test.py

```diff
@@ -43,47 +21,71 @@ def system_spec():
     }
 
 
-class MessageListener(object):
-    create_event_captured = False
+class MessageListener(ConnectionListener):
+    _create_event_captured: bool
+
+    def __init__(self):
+        super().__init__()
+        self._create_event_captured = False
 
     def on_error(self, headers, message):
         print("received an error %s" % headers)
 
-    def on_message(self, headers, message):
-        try:
-            if headers["model_class"] == "Operation":
+    def on_message(self, headers: Dict[str, Any], message: Any) -> None:
+        """Set a property when a MESSAGE frame is received by the STOMP connection.
 
-                parsed = SchemaParser.parse_operation(message, from_string=True)
+        If the needed key is in the headers dict and has the correct value, signal
+        that we've been successful.
 
-                if parsed.model and parsed.model.name:
-                    if parsed.model.name.startswith("REQUEST"):
-                        print(message)
-                        self.create_event_captured = True
+        Args:
+            headers: a dictionary containing all headers sent by the server
+            as key/value pairs.
+
+            message: the frame's payload - the message body.
+        """
+        try:
+            if "model_class" in headers:
+                if headers["model_class"] == "Operation":
+                    parsed = SchemaParser.parse_operation(message, from_string=True)
+
+                    if parsed.model and parsed.model.name:
+                        if parsed.model.name.startswith("REQUEST"):
+                            self.create_event_captured = True
+                        else:
+                            print(f"ERROR: no 'REQUEST' is parsed model")
                     else:
-                        print(parsed.model.name)
-            elif headers["model_class"] == "error_message":
-                print("Error Message Returned:", message)
-        except:
-            print("Error: unable to parse message:", message)
+                        print("ERROR: no parsed model found")
+                elif headers["model_class"] == "error_message":
+                    print(f"ERROR: Message returned: {message!r}")
+                else:
+                    print(f"ERROR: 'model_class' not in headers, message={message}")
+        except Exception:
+            print(f"ERROR: unable to parse, message={message}")
 
+    @property
+    def create_event_captured(self):
+        return self._create_event_captured
 
+    @create_event_captured.setter
```

Comment:
I'm not going to pretend I care all that much, but is there a reason for this?  It doesn't look like anything outside of this class tries to set `_create_event_captured` and inside the class it should be fair game to just set it directly.

Reply:
The reason is that this is the idiom I'm accustomed to and prefer.